### ISSUE TITLE
Announcement and "sister board" bar

### ISSUE THREAD

**Issue by DonaldTsang** [DangerOnTheRanger/maniwani#122]
1. It would be used for semi-hierarchical management of one head board + multiple baby sister boards.
2. It would be used for announcements, news and events of the board (e.g. moderation notice, breaking news of certain topics)
3. see https://8ch.net/fringe and https://8ch.net/rule34

**Comment by samip5** [comment]
Do you have an example idea how this should work, as regarding to global board/site announcements?

**Comment by DonaldTsang** [comment]
Here is something you can do some research on (the Webring protocol for multiple boards)
- https://gitlab.com/alogware/LynxChanAddon-Webring
- https://gitlab.com/Tenicu/infinityaddon-webring
- JSchan
  - https://github.com/fatchan/jschan/blob/master/schedules/webring.js
  - https://github.com/fatchan/jschan/blob/master/db/webring.js

Another implementation idea is to have a notice/announcement bar that allows for board hyperlinking, and we can have a table of community connections between boards. If you want to see an example of the announcement bar see https://julay.world/tech (look at the red text)

**Comment by samip5** [comment]
I was not talking about that, but rather the actual announcements?

**Comment by DonaldTsang** [comment]
@samip5 it is just text really, I don't understand what you would like to know about announcements? It should be set just like board title and description.

**Comment by samip5** [comment]
I haven't looked yet at the current implementation so is there one yet? I did find something to do with alerts, mainly trying to understand what the issue is, only CSS?

**Comment by DonaldTsang** [comment]
@samip5 simple CSS and HTML, there is no JS involed, it is completely cosmetic.

**Comment by samip5** [comment]
Ah, so category of nice to have. :)

**Comment by DonaldTsang** [comment]
@samip5 no, it is a MUST HAVE if you are talking about announcements, "sister boards"? nice to have.

**Comment by samip5** [comment]
Why are you not doing it yourself then?
